src/Facade/Services/ImportBookTransactionCodeService.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace Linn.Stores.Facade.Services
+{
+    using System.Collections.Generic;
+
+    using Linn.Common.Facade;
+    using Linn.Common.Persistence;
+    using Linn.Stores.Domain.LinnApps.ImportBooks;
+
+    public class ImportBookTransactionCodeService : IImportBookTransactionCodeService
```

Comment:
why not just use the standard facade service? then you don't need to write any of this code and we have more functionality available if we need it for free.

Reply:
Yeah should've used it, done now. I think it was because I did the exchange rate one first where I was calling the domain and just didn't think the next ones through properly

Reply:
i remember lewis mapped it as a view in the tax stuff. you could try that for now as we'll never update that table